internal/measurement/measurement.go

```diff
@@ -172,69 +176,55 @@ func Percentage(value, total int64) string {
 	}
 }
 
+// unit includes a list of names representing a specific unit and a factor
+// which one can multiple a value in the specified unit by to get the value
+// in terms of the base unit.
+type unit struct {
+	preferredName string
+	names         []string
+	factor        float64
+}
+
+var memoryUnits = []unit{
+	{"B", []string{"b", "byte"}, 1},
+	{"kB", []string{"kb", "kbyte", "kilobyte"}, 1024},
```

Comment:
It's a bit sad that pprof confuses kB and KiB, MB and MiB and so on. What we have here is IEC binary prefixes, not SI decimal ones...

Reply:
But we are not fixing it now :)

Reply:
Do agree with this!

I was considering if a generic approach to unit conversion could work; but kilo meaning both 10^3 and 2^10 made that less feasible (though, using IEC prefixes in addition to SI prefixes doesn't resolve all issues with the approach I was considering).

I'm not marking this as resolved, because I'd like to switch the canonicalName for these units to  using IEC prefixes as a follow-up (this isn't a complete fix, though).